Test.c

```diff
@@ -4,11 +4,156 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // Test.c
-// Program on branch main
+// Program on branch A4.
+// Program implements test cases, gets user input and checks if its a palindrome
+// Prints if the given input string is a palindrome or if reversed INT overflows
+// Prints the reversed integer if the input is within INT range
 // ------------------------------------------------------------------------------------------------
 
+#pragma warning(disable:4996)
 #include <stdio.h>
+#include <stdbool.h>
+#include "Header.h"
+#include <string.h>
+#include <stdlib.h>
+#include <limits.h>
+#include <conio.h>
+#include <malloc.h>
+#include <ctype.h>
 
-int main () { 
-	printf ("Hello World!\n");
+typedef struct {
+   char* str;

```

Comment:
Please follow naming conventions. Any Property/Field in a struct should start with a capital letter.
```suggestion
   char* String;

```